Automattic/wp-calypso
# Theme Discovery: Implement LiTS Collection View

Related to:
- https://github.com/Automattic/dotcom-forge/issues/4354

We need to implement the LiTS Collection view as specified in the parent issue so that we can release the Theme Discovery project to our users.

## Proposed Changes

* Implemented the designs needed.

Before:
![Captura de pantalla 2023-11-07 a las 14 31 56](https://github.com/Automattic/wp-calypso/assets/1989914/d3905b35-5ded-4b72-9e98-722805b788d5)

After:
![Captura de pantalla 2023-11-07 a las 14 30 48](https://github.com/Automattic/wp-calypso/assets/1989914/287f9c20-0a05-44b1-9c73-2302656ce5a0)

## Testing Instructions



* Open the Calypso live link
* Go to the LiTS and append `flags=themes/discovery-lits`
* Open the collection view of the Partner collection carousel
* You should see the updated style.

## Pre-merge Checklist
